9/community/Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN set -eux; \
     addgroup -S -g 1000 sonarqube; \
     adduser -S -D -u 1000 -G sonarqube sonarqube; \
     apk add --no-cache --virtual build-dependencies gnupg unzip curl; \
-    apk add --no-cache bash su-exec ttf-dejavu openjdk11-jre-headless; \
+    apk add --no-cache bash su-exec ttf-dejavu openjdk11-jre; \
```

Comment:
How about adding `fontconfig` instead?

Reply:
installing `openjdk11-jre-headless` and `fontconfig` is missing `libfontmanager.so` so it would not fix the issue at hand

Reply:
okay